secure_message/repository/saver.py

```diff
@@ -64,3 +64,16 @@ def save_msg_audit(msg_id, msg_urn, session=db.session):
             session.rollback()
             logger.error('Message audit save failed', error=e)
             raise MessageSaveException('Message save failed')
+
+    @staticmethod
+    def save_msg_actors(msg_id, from_actor, to_actor, sent_from_internal, session=db.session):
+        """Save actor information data to database"""
+        actor = Actors(msg_id, from_actor, to_actor, sent_from_internal)
+
+        try:
+            session.add(actor)
+            session.commit()
+        except Exception as e:
+            session.rollback()
+            logger.error('Message actor save failed', error=e)
```

Comment:
This should be a logger.exception, you won't need the as e or to pass the e as an argument.

Reply:
Good call . I copied the existing code , its wrong in the except blocks for all the tables .

Reply:
Why we catching exception and not the relevant sqlalchemy exception?

Reply:
Good call, I will tidy them up . Its the same on all of the tables not just the new one